SmartPlaces.Facilities/README.md

```diff
@@ -0,0 +1,12 @@
+# Microsoft Smart Places & Energy
+
+This repo contains the code, configuration, and documentation for the Microsoft Smart Places Facilities Project. 
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
```

Comment:
Where can the deployment notes be found?

Reply:
Don't have those yet. We'll need to spend some time on the readmes